itou/templates/includes/pagination.html

```diff
@@ -8,11 +8,10 @@
 {% if page.display_pager %}
     {% with request.get_full_path as url %}
         <nav role="navigation"
-            aria-label="Pagination"
-            {% if boost|default:False %}hx-boost="true"{% endif %}
-            {# Don’t specify a default for boost_target and boost_indicator, keeping them required when boost. #}
-            {% if boost|default:False or boost_target %}hx-target="{{ boost_target }}"{% endif %}
-            {% if boost|default:False or boost_indicator %}hx-indicator="{{ boost_indicator }}"{% endif %}>
+             aria-label="Pagination"
+             {% if boost|default:False %}hx-boost="true"{% endif %}
+             {% if boost_target|default="" %}hx-target="{{ boost_target }}"{% endif %}
+             {% if boost_indicator|default="" %}hx-indicator="{{ boost_indicator }}"{% endif %}>
```

Comment:
Cf https://github.com/gip-inclusion/les-emplois/pull/4562#discussion_r1743445696
```suggestion
+            {% if boost|default:False %}hx-boost="true"
+                {% if boost_target %}hx-target="{{ boost_target }}"{% endif %}
+                {% if boost_indicator %}hx-indicator="{{ boost_indicator }}"{% endif %}>
+            {% endif %}
```